online_editor/views.py

```diff
@@ -16,19 +16,46 @@ def index(request):
     context = {}
     return render(request, 'index.html', context)
 
-def execute(request):
-    code = request.body.decode() + "\n" #decoding byte to code
-    
+def run(request):
+    code = request.body.decode() + "\n"
```

Comment:
You can use **_request_content = json.loads(request.body)_** to get the data inside a front-end POST request. The data sent by front-end has a structure: 
For example:
 **_{
  "lang": "python",
  "source": "print 'Hello World'",
  "input": "",
  "memory_limit": 243232,
  "time_limit": 5,
  "id": “213121”,
  "compile_status": True
  }_**

then, you can get each entity by for example:
**_lang = request_content.get("lang")
source = request_content.get("source")_**
and so on. Also, **_memory_limit, time_limit, compile_status, lang, id_** are the data front-end given to back-end, there is no need to fix the value.